src/app/components/App/AppContainer.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import * as firebase from "firebase";
+import { connect } from 'react-redux';
+import { setAuthState } from '../../redux/actions.js';
```

Comment:
We are probably going to need to find a better way to handle pathing before this gets ugly.